addons/website/static/src/snippets/s_dynamic_snippet/000.scss

```diff
@@ -1,11 +1,19 @@
-body:not(.editor_enable) {
-  .s_dynamic {
-    *[data-url] {
-      cursor: pointer;
+body {
+  &:not(.editor_enable) {
+    .s_dynamic {
+      *[data-url] {
+        cursor: pointer;
+      }
+    }
+  }
+  &.editor_enable {
+    .s_dynamic {
+      &> * {
+        pointer-events: none;
+      }
     }
   }
 }
```

Comment:
- 4 spaces indenting please, just to preserve my eyes :) 
- there should not be edit-only scss in standard scss files, the rules should only be there in edit mode -> website.edit_mode.scss is there for that. Unfortunately it won't support our compat system... but if this snippet ever increase its version, we can simply remove the rules associated with edit mode as we don't support the *edition* of deprecated snippets.

Reply:
Sorry did not take the time to setup the linter for this fix... :/
Will do for sure in the future! Too fast too bad!